src/flint/fmpz.jl

```diff
@@ -491,10 +491,6 @@ function rem(x::ZZRingElem, c::ZZRingElem)
   return r
 end
 
-function rem(a::ZZRingElem, b::UInt)
-  return ccall((:fmpz_fdiv_ui, libflint), UInt, (Ref{ZZRingElem}, UInt), a, b)
```

Comment:
as this will remove `ZZ(3) % UInt(2)` I don't like it. Rather do an error if the input is illegal (ie. the remainder negative)